src/assets/images/exchange-dark.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
You should cleanup these svg files with [svggo](https://jakearchibald.github.io/svgomg/)